quarkchain/p2p/peer.py

```diff
@@ -939,31 +940,53 @@ def _add_peer(self, peer: BasePeer, msgs: Tuple[PeerMessage, ...]) -> None:
     async def _cleanup(self) -> None:
         await self.stop_all_peers()
 
-    def blacklist(self, remote_address: Address) -> None:
+    def dialout_blacklist(self, remote_address: Address) -> None:
         # never blacklist bootstap
         for node in self.whitelist_nodes:
             if node.address.ip == remote_address.ip:
                 return
-        self._blacklist[remote_address.ip] = time_ms() // 1000 + BLACKLIST_COOLDOWN_SEC
+        self._dialout_blacklist[remote_address.ip] = time_ms() // 1000 + DIALOUT_BLACKLIST_COOLDOWN_SEC
 
-    def chk_blacklist(self, remote_address: Address) -> bool:
-        if remote_address.ip not in self._blacklist:
+    def chk_dialout_blacklist(self, remote_address: Address) -> bool:
+        if remote_address.ip not in self._dialout_blacklist:
             return False
         now = time_ms() // 1000
-        if now >= self._blacklist[remote_address.ip]:
-            del self._blacklist[remote_address.ip]
+        if now >= self._dialout_blacklist[remote_address.ip]:
+            del self._dialout_blacklist[remote_address.ip]
+            return False
+        return True
+
+    def dialin_blacklist(self, remote_address: Address) -> None:
+        # never blacklist bootstap
+        for node in self.whitelist_nodes:
+            if node.address.ip == remote_address.ip:
+                return
+        self._dialin_blacklist[remote_address.ip] = time_ms() // 1000 + DIALOUT_BLACKLIST_COOLDOWN_SEC
```

Comment:
Suggest use different const name (e.g., DAILIN_BLACKLIST_COOLDOWN_SEC), and I guess they should have different values to improve connectivity :)

Reply:
yeah right, I forgot to change this